t/webservice.t

```diff
@@ -101,17 +101,14 @@ transactionally {
     }
 
     subtest "rename-tag" => {
-        ok my $newTag = $ws.rename-tag($user, '00MyTestTag', '00MyNewTestTag'), "Rename tag";
-        is $newTag.name, '00MyNewTestTag', 'Tag has expected name';
+        ok my $new-name = $ws.rename-tag($user, '00MyTestTag', '00MyNewTestTag'), "Rename tag";
+        is $new-name, '00MyNewTestTag', 'Tag has expected name';
```

Comment:
Well, other than in the test... :-) But still, is it needed.

Reply:
... removed.